### PR TITLE
feat: first version workscheme

### PR DESCRIPTION
**Pull request by sanderdejong88**
# Changes

### CODE REVIEW

File: api/bridge/auth.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    validateHeaders(request, ['authorization']);
+  } catch (e) {
+    return response.status(400).json({
+      message: e.message,
+    });
+  }
+
+  const authCookie = await getAuthCookieValue(request.headers.authorization);
+
+  response.setHeader('Set-Cookie', [`${authCookie}; Path=/;`]);
```
Comment (FrankMerema) [inline api/bridge/auth.ts:19]: Shouldn't we make a TTL or set secure?

File: components/records/weekly-timesheet.vue [hunk 2]
```diff
@@ -205,6 +204,24 @@ export default defineComponent({
...
+        workScheme = await app.$workSchemeService.getWorkScheme({
+          bridgeUid: employee.bridgeUid || '',
+          startDate: new Date(workWeek[0].date),
+          endDate: new Date(workWeek[6].date),
+        });
+        showBridgeError.value = false;
+      } catch ({response}) {
+        if (response.status === 401) {
+          await logout();
+          return;
```
Comment (FrankMerema) [inline components/records/weekly-timesheet.vue:219]: Is this return doing anything?

Comment (sanderdejong88) [reply components/records/weekly-timesheet.vue]: nope..

File: api/bridge/auth.ts (hunk 1 above)
Comment (sanderdejong88) [reply api/bridge/auth.ts]: done

**Review by FrankMerema — APPROVED** [review]

**Pull request merged**